src/spec.c

```diff
@@ -2050,7 +2050,7 @@ static void Indexes_ScanAndReindexTask(IndexesScanner *scanner) {
                     scanner->totalKeys);
   } else {
     RedisModule_Log(ctx, "notice", "Scanning index %s in background: done (scanned=%ld)",
-                    scanner->spec_name, scanner->totalKeys);
+                    scanner->spec_name, scanner->scannedKeys);
```

Comment:
why only here? there are a few more logs that uses the `totalKeys` instead of the `scannedKeys`

Reply:
Didn't look for more places, added them 👍